spec/index.md

```diff
@@ -1298,78 +1298,80 @@ In the reply callback of a [inter-canister method call](#system-api-call), the a
 ### Overview of imports {#system-api-imports}
 
 The following sections describe various System API functions, also referred to as system calls, which we summarize here.
-
-    ic0.msg_arg_data_size : () -> i32;                                          // I U Q CQ Ry CRy F
-    ic0.msg_arg_data_copy : (dst : i32, offset : i32, size : i32) -> ();        // I U Q CQ Ry CRy F
-    ic0.msg_caller_size : () -> i32;                                            // *
-    ic0.msg_caller_copy : (dst : i32, offset: i32, size : i32) -> ();           // *
-    ic0.msg_reject_code : () -> i32;                                            // Ry Rt CRy CRt
-    ic0.msg_reject_msg_size : () -> i32;                                        // Rt CRt
-    ic0.msg_reject_msg_copy : (dst : i32, offset : i32, size : i32) -> ();      // Rt CRt
-
-    ic0.msg_reply_data_append : (src : i32, size : i32) -> ();                  // U Q CQ Ry Rt CRy CRt
-    ic0.msg_reply : () -> ();                                                   // U Q CQ Ry Rt CRy CRt
-    ic0.msg_reject : (src : i32, size : i32) -> ();                             // U Q CQ Ry Rt CRy CRt
-
-    ic0.msg_cycles_available : () -> i64;                                       // U Rt Ry
-    ic0.msg_cycles_available128 : (dst : i32) -> ();                            // U Rt Ry
-    ic0.msg_cycles_refunded : () -> i64;                                        // Rt Ry
-    ic0.msg_cycles_refunded128 : (dst : i32) -> ();                             // Rt Ry
-    ic0.msg_cycles_accept : (max_amount : i64) -> (amount : i64);               // U Rt Ry
-    ic0.msg_cycles_accept128 : (max_amount_high : i64, max_amount_low: i64, dst : i32)
-                           -> ();                                               // U Rt Ry
-
-    ic0.cycles_burn128 : (amount_high : i64, amount_low : i64, dst : i32) -> ();               // I G U Ry Rt C T
-
-    ic0.canister_self_size : () -> i32;                                         // *
-    ic0.canister_self_copy : (dst : i32, offset : i32, size : i32) -> ();       // *
-    ic0.canister_cycle_balance : () -> i64;                                     // *
-    ic0.canister_cycle_balance128 : (dst : i32) -> ();                          // *
-    ic0.canister_status : () -> i32;                                            // *
-    ic0.canister_version : () -> i64;                                           // *
-
-    ic0.msg_method_name_size : () -> i32;                                       // F
-    ic0.msg_method_name_copy : (dst : i32, offset : i32, size : i32) -> ();     // F
-    ic0.accept_message : () -> ();                                              // F
-
-    ic0.call_new :                                                              // U CQ Ry Rt CRy CRt T
-      ( callee_src  : i32,
+The notation `I ∈ {i32, i64}` means that there are actually two corresponding System API functions for the two integer types `I`: `i32` and `i64`.
```

Comment:
```suggestion
The notation `I ∈ {i32, i64}` means that there are actually two corresponding System API functions for the two integer types `I`: `i32` and `i64`.
Since the System API functions target memory 0 of a particular canister, `I` is determined uniformly by the Wasm type of the canister's memory 0.

```